components/back-link/package.json

```diff
@@ -4,7 +4,7 @@
   "dependencies": {
     "@govuk-react/constants": "^0.7.1",
     "@govuk-react/lib": "^0.7.1",
-    "govuk-colours": "^1.0.3"
+   "govuk-colours": "^1.1.0"
```

Comment:
you seem to have removed a space during your find/replace, please can you put it back in? :-)